packages/toplevel_expect_test/toplevel_expect_test.v0.9.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/toplevel_expect_test"
+bug-reports: "https://github.com/janestreet/toplevel_expect_test/issues"
+dev-repo: "git+https://github.com/janestreet/toplevel_expect_test.git"
+license: "Apache-2.0"
+build: [
+  ["jbuilder" "build" "p" name "-j" jobs]
+]
+depends: [
+  "core_kernel"             {>= "v0.9" & < "v0.10"}
+  "findlib_top"             {>= "v0.9" & < "v0.10"}
```

Comment:
oops... this should have been replaced by a constraint on ocamlfind. It's fixed now